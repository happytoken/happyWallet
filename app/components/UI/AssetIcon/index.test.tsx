import React from 'react';
import { shallow } from 'enzyme';
import AssetIcon from './';
const sampleLogo = 'https://hld-chain.oss-cn-beijing.aliyuncs.com/tokens/WBTC.png';

describe('AssetIcon', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<AssetIcon logo={sampleLogo} />);
		expect(wrapper).toMatchSnapshot();
	});
});
