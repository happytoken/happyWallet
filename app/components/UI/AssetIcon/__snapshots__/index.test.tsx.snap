// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AssetIcon should render correctly 1`] = `
<RemoteImage
  fadeIn={true}
  placeholderStyle={
    Object {
      "backgroundColor": "#FFFFFF",
    }
  }
  source={
    Object {
      "uri": "https://hld-chain.oss-cn-beijing.aliyuncs.com/tokens/WBTC.png",
    }
  }
  style={
    Array [
      Object {
        "borderRadius": 25,
        "height": 50,
        "overflow": "hidden",
        "width": 50,
      },
      undefined,
    ]
  }
/>
`;
